tests/geo/test_surface.py

```diff
@@ -558,5 +559,58 @@ def test_convert_back(self):
                     np.testing.assert_allclose(sf1.zs, sf2.zs)
 
 
+class ArclengthTests(unittest.TestCase):
+    def test_arclength_poloidal_angle(self):
+        """
+        Compute arclength poloidal angle from circular cross-section tokamak.
+        Check that this matches parameterization angle.
+        """
+        s = get_surface('SurfaceRZFourier', True, mpol=1, ntor=0,
+                        ntheta=200, nphi=5, full=True)
+        s.rc[0, 0] = 5.
+        s.rc[1, 0] = 1.
+        s.zs[1, 0] = 1.
```

Comment:
I have modified this test so that the minor radius is no longer 1. I also added a test with a rotating ellipse boundary and checked that the angle is in [0,1].